doorstop/core/publisher.py

```diff
@@ -554,7 +561,10 @@ def _table_of_contents_md(obj, linkify=None):
         elif item.header:
             heading = "{h}".format(h=item.header)
         else:
-            heading = item.uid
+            if settings.ENABLE_HEADERS and item.header:
+                heading = item.header
+            else:
+                heading = item.uid
```

Comment:
This section appears to be superfluous now? `ENABLE_HEADERS` is the default in the mainline version of `publisher.py`, i.e., would propose to exclude this section from the patch.